perma_web/perma/templates/archive/single-link.html

```diff
@@ -99,76 +99,96 @@
-                    {% if link.organization %}
-                    <dl class="tray-detail-group">
-                        <dt class="tray-detail-title">Archiving Organization</dt>
-                        <dd class="tray-detail-entry">{{ link.organization }}</dd>
-                    </dl>
-                    {% endif %}
-                {% endcomment %}
-
-                <!--
-                <p class="fp">Perma.cc is a web archiving service developed by the Library Innovation Lab at Harvard University. The data captured in the archived Perma.cc record is the copyright of its original creators. Neither Harvard University nor the archiving individual are responsible for its content. <a href="http://perma.cc/about">More about Perma.cc.</a></p>
-                -->
-
-            </div>
-            <div class="tray-actions">
-
-                {# user edit buttons #}
-                {% if can_toggle_private %}
-                    <button class="btn btn-ui-small btn-dashboard darchive"><span class="_verbose">Mark as </span>{% if link.is_private %}Public{% else %}Private{% endif %}</button>
-                    {% if request.user.is_staff %}
-                        {% if link.is_private %}
-                            (Private reason: {{ link.get_private_reason_display }})
-                        {% else %}
-                            <select name="private_reason">
-                                <option value="user"> At user direction</option>
-                                <option value="policy"> robots.txt or noarchive meta tag</option>
-                                <option value="takedown"> At content owner's request</option>
-                            </select>
-                        {% endif %}
-                    {% endif %}
-                {% endif %}
-
-                {% if can_delete %}
-                    <a class="btn btn-ui-small btn-dashboard user-delete" href="{% url 'user_delete_link' link.guid %}">Delete<span class="_verbose"> record</span></a>
-                    <a class="btn btn-ui-small btn-dashboard user-edit" onclick="return SingleLinkModule.upload_form();">Edit<span class="_verbose"> record</a>
+          <div class="tray-details">
+            <dl class="tray-detail-group">
+              <dt class="tray-detail-title">Source page URL {% if capture.user_upload %}(unverified){% endif %}</dt>
+              <dd class="tray-detail-entry">{{link.submitted_url}}</dd>
+            </dl>
+
+            {% if can_edit %}
+              <dl class="tray-detail-group">
+                <dt class="tray-detail-title">Archive title <span class="save-status"></span></dt>
+                <input name="title" class="col-sm-12 link-submitted-title" value="{{link.submitted_title}}">
+              </dl>
+              <dl class="tray-detail-group">
+                <dt class="tray-detail-title">Notes <span class="label-instruction _verbose"> (only visible to you)</span><span class="save-status"></span></dt>
```

Comment:
Oh, forgot that `{% if link.organization %}`, this is also visible to your org. We should probably include that.